src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -935,6 +935,24 @@ Default to alert color for visibility titles used within an alert context.
     content: attr(data-placeholder);
 }
 
+/* features page */
```

Comment:
Can the changes be moved to after `.remind-no-response`? Otherwise it'll look like `.remind-no-response` is under the features page heading too.